src/SFA.DAS.AssessorService.Api.Types/Models/Certificates/DeleteCertificateRequest.cs

```diff
@@ -0,0 +1,13 @@
+﻿using MediatR;
```

Comment:
Assuming that the Admin branch would be duplicating this code into the Admin service repo where necessary and Shoma knows this?

Reply:
I have updated Shoma on this.